packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -119,32 +120,32 @@ async function convertFHIRBundleToMedicalRecord({
   resources?: ResourceTypeForConsolidation[];
   dateFrom?: string;
   dateTo?: string;
-  conversionType: ConsolidationConversionType;
+  conversionType: MedicalRecordDocType;
 }): Promise<ConversionOutput> {
   const lambdaName = Config.getFHIRToMedicalRecordLambdaName();
 
-  if (!lambdaName) throw new Error("FHIR to Medical Record Lambda Name is undefined");
+  if (!lambdaName) {
+    const msg = "FHIR to Medical Record Lambda Name is undefined";
+    console.log(msg);
```

Comment:
super nit: any particular reason for the log? It should never error, so its NABD to add a log, but it adds to the code (from one line to 5) and the error would bubble up and be reported anyways.

Also, I'd suggest using `out().log` so we keep the pattern to use our logger (w/ req id) - which adds to the changes even more. :)

Reply:
I noticed in the local dev env when I didn't have that env variable, there was no indication in the logs that this was the problem. So it took me a little bit to understand where it went wrong

Reply:
But NBAD, and it is nicer when the code is slightly shorter. So I'll just revert this change